core/schema.py

```diff
@@ -1618,6 +1619,12 @@ def mutate(cls, root, info, **kwargs):
         username = kwargs.get("username")
         password = kwargs.get("password")
         request = info.context
+
+        csrf_middleware = CsrfViewMiddleware(lambda req: None)
+        reason = csrf_middleware.process_view(request, None, (), {})
+        if reason:
+            raise PermissionDenied('CSRF token missing or incorrect.')
```

Comment:
@lruzicki Have you checked how this change affects the direct API calls (e.g. called by 3rd party or even postman?). I'd need confirmation that this change will not break any functioning integrations.

Reply:
Yes, id did work for postman, also on openimis-be there will be option to add allowed origins if any will be blocked.